_episodes/05-loop.md

```diff
@@ -401,7 +401,7 @@ from whatever file is being processed
 > ~~~
 > {: .source}
 > 
-> We need to use
+> Then we would need to write
```

Comment:
I don't really like the use of 'write'. I think 'execute' or 'use' would be better.

Reply:
Any particular reason you prefer those words? I kind of prefer "write" here because we're talking about some surface modifications to the source code we used earlier (adding some double quotes). To my mind, we're still *using* the same tools (a for loop, the `head` command piped to `tail`).

Some other possible wordings to sidestep the issue:

> Then we would need to add double quotes like so:

or

> The code to loop over these files would look like:

Reply:
Sorry for the lack of justification :) The reason I don't like it is the implication that writing (typing) a command is the same as running it.
Your first suggestion sounds good to me. Just something which conveys the message 'we'll need to modify the code thusly'